c-deps/libroach/testutils.h

```diff
@@ -67,8 +67,32 @@ class FakeTimeEnv : public rocksdb::EnvWrapper {
 // 'err_msg' (regexp full match).
 #define EXPECT_ERR(status, err_msg)\
   {\
-    auto s(testutils::compareErrorMessage(status, err_msg)); \
+    auto s(testutils::compareErrorMessage(status, err_msg, false)); \
     EXPECT_TRUE(s.ok()) << s.getState();\
   }
 
+// If err_msg is empty, status must be ok. Otherwise, the status message must match
+// 'err_msg' (regexp full match).
+#define ASSERT_ERR(status, err_msg)\
+  {\
+    auto s(testutils::compareErrorMessage(status, err_msg, false)); \
+    ASSERT_TRUE(s.ok()) << s.getState();\
+  }
+
+// If err_msg is empty, status must be ok. Otherwise, the status message must match
+// 'err_msg' (regexp partial match).
+#define EXPECT_PARTIAL_ERR(status, err_msg)\
```

Comment:
Added partial error matches. This is needed if part of the error is binary (eg: when I dump file contents) as `.` in `std::regex` does not match newlines.